docs/conf/australia/2018/news/thanks-recap.rst

```diff
@@ -0,0 +1,92 @@
+:template: {{year}}/generic.html
+:banner: _static/{{year}}/assets/headers/venue.png
+
+.. post:: Dec 3, 2018
+   :tags: recap, thanks, australia-2018
+
+Highlights and thanks from the Australian conference
+---------------------------------------------------
+
+We had our first Write the Docs Australia conference in Melbourne on 15-16 Nov.
+While we had the Write the Docs Day in 2017, this was our first proper conference that went for 2 days and was
+packed with presentations, workshops, lightning talks and unconference sessions. Here's a quick recap of the 2 days.
+
+Conference recap
+================
+
+At over 100  people, our first conference was a great start to what should now become a regular event on the Write the Docs conference calendar. We had a mix of main stage and lightning talks, unconference sessions, workshops and a social event.
+We received a lot of positive feedback on all of these and look forward to carry them to future events. We also had some interesting feedback on some things we might improve for next time.
+
+- The main stage presentations had a good blend of topics across documentation, user experience, customer support, content strategy, coding and style guides.
+- The unconference sessions and the lightning talks was well received and prompted people to think, collaborate and talk about documentation in a more unstructured way. It also allowed people to discuss topics that were not covered fully or partially throughout the conference.
+- We received a mixed response to the workshops and this can be an area we can look at improving next year.
+Overall, we were so excited to see our first conference come together and see our awesome community gather under one roof.
+
+Conference venue
+================
+
+What better place to hold a documentation conference than a library! We had 2 rooms overlooking the sensational Port Phillip Bay right at the top floor of a public library. While the audience was not soaking in the documentation goodness, they were more than happy to stretch their legs and have a stroll through the library amidst tons of books.
+
+Conscious consumption (aka Food and Drinks)
```

Comment:
This made me double take, and I'm still not quite sure what you mean

Reply:
The library part? Let me revisit it.

Reply:
> Conscious consumption

is what made me :confused:

Reply:
Maybe we could remove it entirely. It was only meant to say "whatever we consumed was with a social consciousness in mind"

Reply:
:+1: following sentence addresses that nicely